www/include/monitoring/status/Services/xml/serviceXML.php

```diff
@@ -141,8 +145,10 @@
     $queryValues['outputToSearch'] = [\PDO::PARAM_STR => '%' . $outputToSearch . '%'];
 }
 
+// TODO: fix order for criticalities
```

Comment:
what does not work in ordering ?

Reply:
Well, it works, I just forgot to remove the comment :D